vcpkg-configuration.json

```diff
@@ -1,7 +1,7 @@
 {
   "default-registry": {
     "kind": "builtin",
-    "baseline": "69efe9cc2df0015f0bb2d37d55acde4a75c9a25b"
+    "baseline": "8b04a7bd93bef991818fc372bb83ce00ec1c1c16"
```

Comment:
I've updated the dependencies on master, so this isn't necessary anymore :)

Reply:
done